dbt_subprojects/hourly_spellbook/models/_metrics/transfers/metrics_transfers_daily.sql

```diff
@@ -0,0 +1,120 @@
+    */
+    select
+        blockchain
+        , block_date
+        , tx_hash
+        , "from" as address
+        , 'sent' as transfer_direction
+        , (sum(amount_usd) * -1) as transfer_amount_usd
+        , count(1) as transfer_count
+    from
+        {{ source('tokens', 'transfers') }}
+    where
+        1 = 1
+    {% if is_incremental() %}
+        and {{ incremental_predicate('block_date') }}
+    {% else %}
+        and block_date >= timestamp '2024-09-01'
+    {% endif %}
+    group by
+        blockchain
+        , block_date
+        , tx_hash
+        , "from"
+        , 'sent'
+
+    union
+
+    select
+        blockchain
+        , block_date
+        , tx_hash
+        , to as address
+        , 'received' as transfer_direction
+        , sum(amount_usd) as transfer_amount_usd
+        , count(1) as transfer_count
+    from
+        {{ source('tokens', 'transfers') }}
+    where
+        1 = 1
+    {% if is_incremental() %}
+        and {{ incremental_predicate('block_date') }}
+    {% else %}
+        and block_date >= timestamp '2024-09-01'
+    {% endif %}
+    group by
+        blockchain
+        , block_date
+        , tx_hash
+        , to
+        , 'received'
+), transfers_amount as (
+    /*
+    - create one column for transfer amount received, one column for transfer amount sent
+    */
+    select 
+        blockchain
+        , block_date
+        , tx_hash
+        , address
+        , sum(case when transfer_direction = 'sent' then transfer_amount_usd else 0 end) as transfer_amount_usd_sent
+        , sum(case when transfer_direction = 'received' then transfer_amount_usd else 0 end) as transfer_amount_usd_received
+        , sum(transfer_count) as transfer_count
+    from
+        raw_transfers     
+    group by
+        blockchain
+        , block_date
+        , tx_hash
+        , address
+), net_transfers as (
+    /*
+    - add amount received and amount sent (since transfer amount sent is set to a negative number, this calculates the net transfer amount received)
+    - for any given address in a single transaction
+    */
+    select
+        blockchain
+        , block_date
+        , tx_hash
+        , address
+        , sum(coalesce(transfer_amount_usd_sent, 0)) as transfer_amount_usd_sent
+        , sum(coalesce(transfer_amount_usd_received, 0)) as transfer_amount_usd_received
+        , sum(coalesce(transfer_amount_usd_received, 0)) + sum(coalesce(transfer_amount_usd_sent, 0)) as transfer_amount_usd
+        , sum(transfer_count) as transfer_count
+    from
+        transfers_amount
+    group by
+        blockchain
+        , block_date
+        , tx_hash
+        , address
+)
+select 
+    blockchain
+    , block_date
+    , sum(transfer_amount_usd_sent) as transfer_amount_usd_sent
+    , sum(transfer_amount_usd_received) as transfer_amount_usd_received
+    , sum(transfer_amount_usd) as transfer_amount_usd
```

Comment:
- Rename transfer_amount_usd to net_transfer_amount_usd
- remove the transfer count field